src/modules/75-cf/utils/UsageLimitUtils.ts

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2022 Harness Inc. All rights reserved.
+ * Use of this source code is governed by the PolyForm Shield 1.0.0 license
+ * that can be found in the licenses directory at the root of this repository, also available at
+ * https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt.
+ */
+
+import { BannerType } from '@common/layouts/Constants'
+import { formatToCompactNumber } from '@cf/utils/CFUtils'
+import type { UseStringsReturn } from 'framework/strings'
+
+interface BannerText {
+  message: string | undefined
```

Comment:
You can just mark this as optional. Does the same thing
```suggestion
  message?: string
```

Reply:
done